.github/workflows/build-connector-images-command.yml

```diff
@@ -65,11 +65,23 @@ jobs:
           # Use fetch-depth: 0 to ensure we can access the full commit history
           fetch-depth: 0
 
+      - name: Install uv
+        uses: astral-sh/setup-uv@d0cc045d04ccac9d8b7881df0226f9e82c39688e # v6.8.0
+
       - name: Generate Connector Matrix from Changes
         id: generate-matrix
         run: |
-          # Get the list of modified connectors
-          echo "connectors_matrix=$(./poe-tasks/get-modified-connectors.sh --json)" | tee -a $GITHUB_OUTPUT
+          # Get the list of modified connectors using airbyte-ops CLI
+          MATRIX_JSON=$(uvx --from airbyte-internal-ops airbyte-ops repo connector list . \
+            --modified-only \
+            --output-format json-gh-matrix)
+
+          # Transform from {"include": [...]} to {"connector": [...]} format for compatibility
+          if [ "$(echo "$MATRIX_JSON" | jq '.include | length')" -eq 0 ]; then
+            echo "connectors_matrix={\"connector\": [\"\"]}" | tee -a $GITHUB_OUTPUT
+          else
+            echo "connectors_matrix=$(echo "$MATRIX_JSON" | jq '{connector: [.include[].connector]}')" | tee -a $GITHUB_OUTPUT
+          fi
```

Comment:
The output should already in the correct JSON format to be passed directly as the matrix input. If not, it's a bug upstream and we should fix it.